pkg/printers/internalversion/printers_test.go

```diff
@@ -1927,7 +1927,7 @@ func TestTranslateTimestampSince(t *testing.T) {
 		{"unknown", translateTimestampSince(metav1.Time{}), "<unknown>"},
 		{"30 seconds ago", translateTimestampSince(metav1.Time{Time: time.Now().Add(-3e10)}), "30s"},
 		{"5 minutes ago", translateTimestampSince(metav1.Time{Time: time.Now().Add(-3e11)}), "5m"},
-		{"an hour ago", translateTimestampSince(metav1.Time{Time: time.Now().Add(-6e12)}), "1h"},
+		{"an hour ago", translateTimestampSince(metav1.Time{Time: time.Now().Add(-6e12)}), "100m"},
```

Comment:
Should this not be "60m" instead of "100m" ? :)

Reply:
I think the textual representation gets "truncated" down